src/components/StatusColorOptionsEditor.tsx

```diff
@@ -1,37 +1,49 @@
-import { ColorDefinition, PanelOptionsEditorProps, getColorForTheme, standardEditorsRegistry } from '@grafana/data';
-import { HorizontalGroup, Label, getTheme } from '@grafana/ui';
+import { PanelOptionsEditorProps } from '@grafana/data';
+import { ColorPicker, HorizontalGroup, Label } from '@grafana/ui';
 import { StatusPanelOptions } from 'lib/statusPanelOptionsBuilder';
+import { config } from '@grafana/runtime';
+
 import React from 'react';
 
-export const StatusColorOptionsEditor: React.FC<PanelOptionsEditorProps<StatusPanelOptions['colors']>> = ({
-  value,
-  onChange,
-}) => {
+export const StatusColorOptionsEditor: React.FC<PanelOptionsEditorProps<StatusPanelOptions['colors']>> = ({value, onChange}) => {
   // get grafana color picker and theme
-  const colorPicker = standardEditorsRegistry.get('color').editor as any;
-  const theme = getTheme();
+  // const colorPicker = standardEditorsRegistry.get('color').editor as any;
+  // const theme = getTheme();
 
   // helper function to build the handler for each color
-  const buildHandler = (key: keyof StatusPanelOptions['colors']) => (color: ColorDefinition) =>
-    onChange({ ...value, [key]: getColorForTheme(color, theme as any) });
+  // const buildHandler = (key: keyof StatusPanelOptions['colors']) => (color: ColorDefinition) =>
+  //   onChange({ ...value, [key]: getColorForTheme(color, theme as any) });    
+
+  
+
+  const buildHandler = (key: keyof StatusPanelOptions['colors']) => (color: string) => {
+    onChange({
+      ...value,
+      [key]: color,
+    });
+  };
 
   return (
     <HorizontalGroup spacing="lg">
       <div>
         <Label>Critical</Label>
-        {colorPicker({ value: value.crit, onChange: buildHandler('crit') })}
+        <ColorPicker color={value?.crit ||config.theme2.visualization.getColorByName("red")} onChange={buildHandler('crit')} />
```

Comment:
Instead of accessing the theme via the config you should use the following hook to get access to it in your component:
```ts
import { useTheme2 } from '@grafana/ui';

export const StatusColorOptionsEditor: React.FC<PanelOptionsEditorProps<StatusPanelOptions['colors']>> = ({value, onChange}) => {
   const theme = useTheme2();
   // ....
}
```